flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolUtils.java

```diff
@@ -45,19 +50,108 @@
  */
 public class SlotPoolUtils {
 
+	public static final Time TIMEOUT = Time.seconds(10L);
+
 	private SlotPoolUtils() {
 		throw new UnsupportedOperationException("Cannot instantiate this class.");
 	}
 
+	public static SlotPoolBuilder createSlotPoolBuilder() {
```

Comment:
These `createSlotPoolBuilder(...)` does not how much benefit to me, given that in those tests there are already reusable methods to create and setup slot pool.

Reply:
I've removed these `createSlotPoolBuilder` functions.